src/dataflow/source/kafka.rs

```diff
@@ -329,9 +329,12 @@ where
                                     );
                                 };
 
-                                let out = payload.to_vec();
+                                let out = message.payload().map(|p| p.to_vec()).unwrap_or_default();
```

Comment:
Can you leave a comment explaining when payloads can be null? (Or at least, retain the one about the Debezium case).

So future readers will understand why it is okay to replace a null payload with an empty vec.